src/Microsoft.Tye.Hosting/Model/DockerRunInfo.cs

```diff
@@ -15,6 +16,7 @@ public DockerRunInfo(string image, string? args)
         }
 
         public bool Private { get; set; }
+        public bool IsAspNet => Image.StartsWith("mcr.microsoft.com/dotnet/core/aspnet", StringComparison.OrdinalIgnoreCase);
```

Comment:
Is this the only way to determine ASPNET? If #240 is supported then this fails as a check allowing devs to create their own base images.

Reply:
> Is this the only way to determine ASPNET?

yeah this seems backwards. We *already know* that this came from an ASP.NET Core project. We throw away that knowledge and use a heuristic that's easily broken - improving that heuristic would take a lot of work - we should just try to propagate the knowledge that it was an ASP.NET Core project originally.

Reply:
I cheated I can change it though. I was also looking at our backyard tests 😬